docs/essentials/backgrounds.mdx

```diff
@@ -63,29 +63,51 @@ You don't need additional configuration to get started. But its properties are f
 
 {/* prettier-ignore-end */}
 
-### Disable the grid
+## Setting value
 
-If you need to disable the grid for a specific story, set the `backgrounds` parameter to the following:
+You can set the background color in the toolbar by clicking on the color you want to use.
 
-{/* prettier-ignore-start */}
+If you want to enforce a story to use a specific background color, you can do so by setting the `backgrounds` global in the story:
 
-<CodeSnippets path="storybook-addon-backgrounds-disable-grid.md" usesCsf3 csf2Path="essentials/backgrounds#snippet-storybook-addon-backgrounds-disable-grid" />
+```ts filename="Button.stories.ts|tsx" renderer="common" language="ts-4-9"
+// Replace your-framework with the name of your framework
+import type { Meta, StoryObj } from '@storybook/your-framework';
 
-{/* prettier-ignore-end */}
+import { Button } from './Button';
+
+const meta = {
+  component: Button,
+  parameters: {
+    backgrounds: {
+      options: {
+        maroon: { name: 'Maroon', value: '#400' },
+      },
+    },
+  },
+
+} satisfies Meta<typeof Button>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Large: Story = {
+  global: {
+    backgrounds: { value: 'maroon', grid: false },
+  },
+};
+```
 
 ## API
 
 ### Parameters
 
 This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `backgrounds` namespace:
 
-#### `default`
-
-Type: `string`
+#### `options`
 
-Default background color. Must match the `name` property of one of the [available colors](#values).
+Type: `Record<string, { name: string; value: string }>`
```

Comment:
Should name be optional?

Reply:
Do you have a reason for making it optional?

Reply:
So that it can default to the key if unset?

Reply:
But why? Why have 2 ways to configure things?
By just having 1 way, it's clearer for us, clearer for users, less docs to write, less complexity in code, less potential for bugs.

I think we should enforce there to be a name prop, that's purely presentational and separate from the identifier.
If we support both, there's a good chance users won't even know about the name prop at all, and try to hack things in the key instead. Which could cause bugs, with character escaping.

I strongly believe we should strive for simplicity, rather than terseness.